testing/dart/color_test.dart

```diff
@@ -50,9 +50,9 @@ void main() {
   test('two colors are only == if they have the same runtime type', () {
     expect(const Color(0x12345678), equals(const Color(0x12345678)));
     expect(const Color(0x12345678), equals(Color(0x12345678))); // ignore: prefer_const_constructors
-    expect(const Color(0x12345678), isNot(equals(const Color(0x87654321))));
-    expect(const Color(0x12345678), isNot(equals(const NotAColor(0x12345678))));
-    expect(const NotAColor(0x12345678), isNot(equals(const Color(0x12345678))));
+    expect(const Color(0x12345678), notEquals(const Color(0x87654321)));
+    expect(const Color(0x12345678), notEquals(const NotAColor(0x12345678)));
+    expect(const NotAColor(0x12345678), notEquals(const Color(0x12345678)));
     expect(const NotAColor(0x12345678), equals(const NotAColor(0x12345678)));
```

Comment:
FWIW, I'm not a fan of these kind of matchers anyway. I'd much rather just see `expect(Color(...) == Color(...), true);`, or even just make the second parameter to expect optional so that it more or less workslike `assert`. 

That said, I like `notEquals` better than `isNot(somethingElse(isNot(.....`.